v2/ordered-changestream-buffer-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/processing/handler/InputRecordProcessor.java

```diff
@@ -38,7 +40,7 @@ public class InputRecordProcessor {
   public static void processRecords(
       List<String> recordList,
       Schema schema,
-      MySqlDao dao,
+      PostgreSqlDao dao,
```

Comment:
Have a new base class here rather than an implementation. Please avoid code duplication.

Reply:
I have push a commit now to address this. Please review . Thank you for the feedback